src/nostr_service.cpp

```diff
@@ -517,8 +513,9 @@ void NostrService::disconnect(string relay)
 
 string NostrService::generateSubscriptionId()
 {
-    uuid uuid = random_generator()();
-    return to_string(uuid);
+    UUIDv4::UUIDGenerator<std::mt19937_64> uuidGenerator;
+    UUIDv4::UUID uuid = uuidGenerator.getUUID();
+    return uuid.bytes();
```

Comment:
Can we add `using UUIDv4::UUIDGenerator` and `using UUIDv4::UUID` to the usings block at the top of this file?  That will keep the code a bit more readable in-line, IMO.